data/kahet/beyond patir assets.txt

```diff
@@ -0,0 +1,2471 @@
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -2
+		"lifetime" 15
+		"random lifetime" 5
+		"submunition" "wep static"
+		"submunition" "wep37 print"
+	
+outfit "wep37 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -2
+		"lifetime" 15
+		"random lifetime" 5
+		"submunition" "wep static"
+		"submunition" "wep38 print"
+	
+outfit "wep38 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -2
+		"lifetime" 15
+		"random lifetime" 5
+		"submunition" "wep static"
+		"submunition" "wep39 print"
+	
+outfit "wep39 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -2
+		"lifetime" 15
+		"random lifetime" 5
+		"submunition" "wep static"
+		"submunition" "wep40 print"
+	
+outfit "wep40 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -2
+		"lifetime" 15
+		"random lifetime" 5
+		"submunition" "wep static"
+		"submunition" "wep41 print"
+	
+outfit "wep41 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -2
+		"lifetime" 15
+		"random lifetime" 5
+		"submunition" "wep static"
+		"submunition" "wep42 print"
+	
+outfit "wep42 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -2
+		"lifetime" 15
+		"random lifetime" 5
+		"submunition" "wep static"
+
+outfit "wep static"
+	weapon
+		sprite "effect/asteroid projectile static"
+			"start frame" 1
+			"frame rate" 9
+		"acceleration" .0000001
+		"drag" 1
+		"lifetime" 575
+		"turn" 360
+		"homing" 2
+		"tracking" 1
+		"trigger radius" 240
+		"blast radius" 400
+		"discharge damage" 1.7
+		"hull damage" 72.9
+		"slowing damage" 10
+		"hit force" 150
+		"hit effect" "circle defense"
+
+
+effect "circle defense"
+	sprite "effect/explosion/nuke"
+		"frame time" 60
+		"random start frame"
+	sound "point defense"
+	lifetime 2
+
+# Invisible ship in Lathia, represents the power of the asteroid base in the system
+
+mission "Asteroid guarding Lathia"
+	invisible
+	landing
+	to complete
+		never
+	npc save
+		government "Magic Asteroid"
+		personality forbearing frugal heroic staying vindictive
+		system Lathia
+		ship "Asteroid Planet" "Queen Asteroid"
+
+ship "Asteroid Planet"
+	attributes
+		category "Heavy Warship"
+		"uncapturable" 1
+		"inscrutable" 1
+		"cost" 1350000000
+		"hull" 86135672193
+		"required crew" 1
+		"bunks" 1
+		"mass" 674
+		"drag" 30.4
```

Comment:
Shouldn't these be much higher given the nominal size of this thing?

Or, if it is actually this size, it shouldn't have 86 times more hull than an Archon.

Reply:
I had chosen these values more to give the right look when moving (appearing somewhat inertialess, reaching the maximum speed almost instantly in any direction), but this can be easily kept by multiplying thrust and turn by the same amount; so, I've multiplied all these by a factor of 10